tests/phpunit/tests/theme/wpAddGlobalStylesForBlocks.php

```diff
@@ -334,25 +331,4 @@ private function get_global_styles() {
 		$actual = wp_styles()->get_data( 'global-styles', 'after' );
 		return is_array( $actual ) ? $actual : array();
 	}
-
-	/**
-	 * Get cache key for `wp_styles_for_blocks`.
-	 *
-	 * @return string The cache key.
-	 */
-	private function get_wp_styles_for_blocks_cache_key() {
```

Comment:
Curious, why remove this test rather than adjust it?

Reply:
It's no longer needed after the changes. This is a utility method to make it easier to generate the previous cache key value that is no longer needed.